.gitignore

```diff
@@ -16,3 +16,9 @@ test_crates/**/Cargo.lock
 
 # ide
 .idea
+
+# bench
+flamegraph.svg
+perf.data
+perf.data.old
+benches/aws-sdk-ec2.json
```

Comment:
Would it be possible to place this inside e.g. `localdata/benches/` instead?

`localdata/` is where the tests' rustdoc JSONs are stored, so the symmetry would be nice there.

Reply:
`flamegraph.svg` and `perf.data*` are dumped in the CWD, but I moved the `aws-sdk-ec2.json`